sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -348,6 +348,21 @@ task validatesContainer() {
   }
 }
 
+task validatesContainerARM() {
+  def pyversion = "${project.ext.pythonVersion.replace('.', '')}"
+  dependsOn 'initializeForDataflowJob'
+  dependsOn ":sdks:python:container:py${pyversion}:docker"
+  def runScriptsPath = "${rootDir}/sdks/python/container/run_validatescontainer.sh"
```

Comment:
as mentioned in the other comment, I think it would be better to configure the script to run in  ARM mode around here.

Reply:
Done.